src/iterators/SingleCellIteratorWrapper.h

```diff
@@ -0,0 +1,78 @@
+/**
+ * SingleCellIteratorWrapper.h
+ *
+ *  Created on: 31 May 2018
+ *      Author: seckler
+ */
+#pragma once
+
+#include <utils/ExceptionHandler.h>
+#include "SingleCellIteratorInterface.h"
+
+namespace autopas {
+
+/**
+ * ParticleIteratorInterface class to iterate over particles.
```

Comment:
Wrong doc. Here also the necessity for a wrapper should be explained in a way someone new to the code understands it.

Reply:
fixed